src/pages/LikedFoodPage.js

```diff
@@ -122,7 +122,7 @@ const LikedFoodPageStyle = styled.section`
 
 const Header = styled.header`
   background-color: var(--lightsteel);
-  padding: 1rem;
+  padding: 1rem 1rem 0 1rem;
   width: 100%;
```

Comment:
hey tanja! really nice work so far!!! just little tip...you could even write padding shorter...`padding: 1rem 1rem 0; /* top, right & left, bottom */` but it´s not a gamechanger :D